pyupgrade.py

```diff
@@ -1535,7 +1539,15 @@ def visit_Call(self, node: ast.Call) -> None:
         elif self._is_six(node.func, ('b', 'ensure_binary')):
             self.six_b.add(_ast_to_offset(node))
         elif self._is_six(node.func, SIX_CALLS) and not _starargs(node):
-            self.six_calls[_ast_to_offset(node)] = node
+            needs_parenthesis = (
+                node.args and
+                isinstance(node.args[0], ast.BoolOp)
+            )
+            six_calls = (
+                self.six_calls_parenthesized
+                if needs_parenthesis else self.six_calls
+            )
+            six_calls[_ast_to_offset(node)] = node
```

Comment:
please write an if statement instead of this indirection -- it'll be much more readable